src/js/client/main.js

```diff
@@ -87,8 +87,11 @@ $(document).ready(function () {
     $(window).scroll(() => {
       if (resize) {
         window.requestAnimationFrame(() => {
-          document.querySelector('.landing-welcome-background-box1').style.setProperty('--diamond-scale', `${1 + window.scrollY / 1000}`);
-          resize = true;
+          const diamond = document.querySelector('.landing-welcome-background-box1');
```

Comment:
Yeah, I guess we could prefer this. To me, it seems fairly balanced between the additional complexity of the expression, versus the elegance. This is where a Swift-style optional chaining syntax would be really nice...